screens/ProductMainScreen.js

```diff
@@ -10,17 +10,60 @@ import SortByMenu from '../components/DropDownMenu/SortByMenu';
 import StoresMenu from '../components/DropDownMenu/StoresMenu';
 import SnackBar from '../components/SnackBar';
 import ProductMainCard from '../components/ProductMainCard';
+import SearchBar from '../components/SearchBar';
 
 export default function ProductMainScreen({ navigation }) {
+  // For account button to open drawer navigator
+  // const navigation = useNavigation();
+  const [searchRadius, setSearchRadius] = useState('499');
+  const [posts, setPosts] = React.useState([]);
+  const [activeTags, setActiveTags] = React.useState([]);
+  const [sort, setSort] = React.useState('Sort: Rating');
   const [refineModalVisible, setRefineModalVisibility] = useState(false);
   const showRefineModal = () => setRefineModalVisibility(true);
   const hideRefineModal = () => setRefineModalVisibility(false);
   const handleRefineModalPress = () => {
     showRefineModal();
   };
 
-  // Default 5km for the placeholder
-  const [distance, setDistance] = useState('');
+  const loadData = async () => {
+    const searchUri = `http://localhost:8000/posts?latitude=5.2&longitude=4.3&radius=${searchRadius}000${activeTags.map(
+      (tag) => `&tag=${tag}`
+    )}`;
+    const apiData = await fetch(searchUri);
```

Comment:
I ran the backend on port 8000 and started the front end. The product main screen does not have any product post show up neither on my iphone nor android stimulator. I tried to console log the error.

` const apiData = await fetch(searchUri).catch((error) => {
      console.error(error);
    });`

See screenshots below: 

![Screen Shot 2020-06-24 at 8 44 31 PM](https://user-images.githubusercontent.com/38272570/85651088-e2b4cb80-b65b-11ea-88d1-c07d05829756.png)
![Screen Shot 2020-06-24 at 8 44 23 PM](https://user-images.githubusercontent.com/38272570/85651094-e5172580-b65b-11ea-9636-bf2766183b64.png)